src/documents/Panels.md

```diff
@@ -231,8 +231,8 @@ fluid.defaults("demo.panels.language", {
     protoTree: {
```

Comment:
Not part of this work, but we could correct a typo here on line 63
````
Preference Maps are **only** required if you are working with
[schemas](PrimarySchemaForPreferencesFramework.md))
````
there is an extra ) after the link here